src/test/kotlin/org/rustPerformanceTests/RsBuildDefMapTest.kt

```diff
@@ -31,7 +31,7 @@ class RsBuildDefMapTest : RsRealProjectTestBase() {
     fun `test build juniper`() = doTest(JUNIPER)
 
     private fun doTest(info: RealProjectInfo) {
-        DefMapService.setUseNewResolve(project, testRootDisposable)
+        DefMapService.setNewResolveEnabled(project, testRootDisposable, true)
```

Comment:
Let's use `UseNewResolve` instead not to use two different ways to do something in different places